sos/report/plugins/process.py

```diff
@@ -21,7 +21,9 @@ class Process(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, CosPlugin):
         ("lsof", "gathers information on all open files", "slow", True),
         ("lsof-threads", "gathers threads' open file info if supported",
          "slow", False),
-        ("smaps", "gathers all /proc/*/smaps files", "", False)
+        ("smaps", "gathers all /proc/*/smaps files", "", False),
+        ("iotop", "gathers information from iotop, by default 20 "
+            "snapshots within 0.5 seconds", "slow", True)
```

Comment:
I think rather than toggling the run of `iotop`, we should be allowing users to control the number of samples to take, e.g. `sos report -k process.samples=50` or `sos report -k process.samples=0` the latter of which effectively gracefully disables the command anyways:

```
[jake@terra ~]$ sudo iotop -b -o -n 0 -d 0.5 -t
[jake@terra ~]$ 

```

Reply:
Change done, now it captures by default 20 samples but you can modify the number in the command line with -k process.samples. I've expanded the commit message as well to reflect this, I hope is the right way to do it.